diesel_derives/src/identifiable.rs

```diff
@@ -14,12 +14,13 @@ pub fn derive(item: syn::DeriveInput) -> Result<quote::Tokens, Diagnostic> {
     ref_generics.params.push(parse_quote!('ident));
     let (ref_generics, ..) = ref_generics.split_for_impl();
 
-    let (field_ty, field_access): (Vec<_>, Vec<_>) = model
+    let t: (Vec<_>, Vec<_>) = model
```

Comment:
Why the temporary variable here? `let (ref field_ty, ref field_access)` should work fine.